### PR TITLE
Scan: Remove redundant emptyContainer styles

### LINKED ISSUE #2614: Remove redundant empty container styles
Right now, there are a lot of `Styles.emptyContainer`. Let's create `EmptySection` that receives `bgColor`, `height` as props.

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #2614 

## Implementation details
- Remove redundant emptyContainer styles

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 ### Scan
 
+- (impv) [\#2713](https://github.com/bandprotocol/bandchain/pull/2713) Remove redundant emptyContainer styles
```
Comment (evilpeach) [inline CHANGELOG_UNRELEASED.md]: Let's use `chore`

Comment (evilpeach) [inline CHANGELOG_UNRELEASED.md]: Let's use `chore`

File: CHANGELOG_UNRELEASED.md [hunk 2]
```diff
@@ -25,6 +25,7 @@
...
 - (impv) [\#2695](https://github.com/bandprotocol/bandchain/pull/2695) Remove unused images and replace with icon.
```
Comment (evilpeach) [inline CHANGELOG_UNRELEASED.md]: Change to `chore` as well

File: scan/src/components/TotalRequestsGraph.re [hunk 3]
```diff
@@ -209,7 +199,7 @@ let make = () => {
        let show = data->Belt.Array.size > 5;
        <div className=Styles.innerCard>
          <canvas id="historicalRequest" className={Styles.chart(show)} />
-         <div className={Styles.emptyContainer(!show)}>
+         <EmptyContainer display={!show ? `flex : `none} height={`percent(100.)}>
```
Comment (evilpeach) [inline scan/src/components/TotalRequestsGraph.re]: Let's make `hide` as props.

File: scan/src/components/HistoricalBondedGraph.re [hunk 4]
```diff
@@ -165,7 +155,7 @@ let make = (~operatorAddress) => {
     <div className=Styles.container>
       <canvas id="historicalBonded" className={Styles.chart(show)} />
       {!show
-         ? <div className=Styles.emptyContainer>
+         ? <EmptyContainer height={`percent(100.)}>
```
Comment (evilpeach) [inline scan/src/components/HistoricalBondedGraph.re]: Make this like `TotalRequestGraph`. Using `hide` props  instead.

File: scan/src/reusable/EmptyContainer.re [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+[@react.component]
+let make =
+    (
+      ~height=`px(300),
+      ~display=`flex,
+      ~justifyContent=`center,
+      ~alignItems=`center,
+      ~flexDirection=`column,
```
Comment (evilpeach) [inline scan/src/reusable/EmptyContainer.re]: 1. It's no need to pass flex attributes. 
2. pass `hide` attribute

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — COMMENTED** [review]
LGTM @pzshine Please review this.

**Review by evilpeach — APPROVED** [review]

**Review by pzshine — APPROVED** [review]
LGTM as well!

**Pull request merged**